pkg/service/auth.go

```diff
@@ -51,6 +52,7 @@ var (
 	statusCodeMapping = map[rpc.Code]envoy_type.StatusCode{
 		rpc.OK:                  envoy_type.StatusCode_OK,
 		rpc.FAILED_PRECONDITION: envoy_type.StatusCode_BadRequest,
+		rpc.INVALID_ARGUMENT:    envoy_type.StatusCode_BadRequest,
 		rpc.NOT_FOUND:           envoy_type.StatusCode_NotFound,
 		rpc.UNAUTHENTICATED:     envoy_type.StatusCode_Unauthorized,
 		rpc.PERMISSION_DENIED:   envoy_type.StatusCode_Forbidden,
```

Comment:
I would imagine it's implemented as a hashmap on the heap, but really not sure.

Otherwise, the compiler would have to ensure there's no writing to it despite declared as `var`, but I guess offering `var` and `const` is precisely the way it has to say "it's your problem, not mine."

My only comfort is that it's a map of known size upfront. Performance should be acceptable (somewhat near _O_(1) in Golang.) Of course, not better than a lookup at compilation time.